plugins/chain_plugin/chain_plugin.cpp

```diff
@@ -360,7 +360,7 @@ void chain_plugin::set_program_options(options_description& cli, options_descrip
 #endif
          ("enable-account-queries", bpo::value<bool>()->default_value(false), "enable queries to find accounts by various metadata.")
          ("max-reversible-blocks", bpo::value<uint32_t>()->default_value(config::default_max_reversible_blocks),
-          "Approximate maximum allowed reversible blocks before shutdown. Will shut down if limit reached.")
+          "Approximate maximum allowed reversible blocks before shutdown. Will shut down if limit reached. Specify 0 to disable.")
```

Comment:
I don't see where you set it to an internal big number to disable if the user configures it to 0.

Reply:
https://github.com/AntelopeIO/spring/pull/493/files#diff-42e9f97eed543dd784b1b77a536c50ff8e2403d5bbd3b1fb7f5dd90201391061R4717

Reply:
Now that I'm not adding the value the `reversible_block_num`, might be nicer to just set it to `std::numeric_limits<uint32_t>::max()` if `0`.

Reply:
Ah, but the `if > 0` does avoid the mutex if that has any worth. I guess I'll leave it as is.